pypesto/visualize/sampling.py

```diff
@@ -91,7 +117,733 @@ def sampling_fval_trace(
     return ax
 
 
-def sampling_parameters_trace(
+def _get_level_percentiles(level: float) -> Tuple[float, float]:
+    """Convert a credibility level to percentiles.
+
+    Similar to highest-density regions of a normal distribution.
```

Comment:
you mean that for a symmetric single-modal distribution, this is a HDI? Maybe reformulate then, didn't get the context on first reading.